src/org/labkey/test/util/DataRegionTable.java

```diff
@@ -997,9 +998,19 @@ public void removeColumn(String columnName, boolean errorExpected)
 
         if (errorExpected)
         {
-            Window removeError = new Window("Error", getDriver());
-            assertTrue(removeError.getBody().contains("You must select at least one field to display in the grid."));
-            removeError.clickButton("OK", true);
+            // If Ext4 is not on the page when Remove Column is clicked, this error will show as a bootstrap modal (i.e. LabKeyAlert)
+            LabKeyAlert alert = LabKeyAlert.getFinder(getDriver()).find();
+            if (alert != null)
+            {
+                assertTrue(alert.getText().contains("You must select at least one field to display in the grid."));
+                alert.accept();
```

Comment:
I think this should wait for a page load (assuming it behaves the same as the Ext4 dialog)
```suggestion
                doAndWaitForPageToLoad(() -> alert.accept());
```

Reply:
I don't believe the page load happens when this dialog closes for the "remove final column error case". It is just canceling the action from what I can tell in testing. 

The `true` param for the `removeError.clickButton()` Ext4 case is saying it should wait for the dialog to close (nothing to do with reload from what I can see. @labkey-tchad did you see some where in this code which looks to be checking for page load?

Reply:
As you surmised, I was misinterpreting that `true` parameter. Thank you.